Syncing with Main branch

htippanaboya opened the pull request:
## ARM API Information (Control Plane)

<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow.
</i>
Azure 1st Party Service can try out the [Shift Left](https://aka.ms/ShiftLeft) experience to initiate API design review from ADO code repo.  If you are interested, may request engineering support by filling in with the form https://aka.ms/ShiftLeftSupportForm.

### Changelog
Add a changelog entry for this PR by answering the following questions:
  1. What's the purpose of the update?
      - [ ] new service onboarding
      - [ ] new API version
      - [ ] update existing version for new feature
      - [ ] update existing version to fix swagger quality issue in s360
      - [ ] Other, please clarify
  2. When are you targeting to deploy the new service/feature to public regions? Please provide the date or, if the date is not yet available, the month.
  3. When do you expect to publish the swagger? Please provide date or, the the date is not yet available, the month.
  4. By default, Azure SDKs of all languages (.NET/Python/Java/JavaScript for both management-plane SDK and data-plane SDK, Go for management-plane SDK only ) MUST be refreshed with/after swagger of new version is published. If you prefer NOT to refresh any specific SDK language upon swagger updates in the current PR, please leave details with justification here.

### Contribution checklist (MS Employees Only):
- [ ] I commit to follow the [Breaking Change Policy](https://aka.ms/AzBreakingChangesPolicy) of "no breaking changes"
- [ ] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [ ] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist

> **Applicability**: :warning:
>
> If your changes encompass only the following scenarios, you should SKIP this section, as these scenarios do not require ARM review.
> - Change to data plane APIs
> - Adding new properties
> - All removals

Otherwise your PR may be subject to ARM review requirements. Complete the following:
- [ ] Check this box if any of the following apply to the PR so that the label "ARMReview" and "WaitForARMFeedback" will be added by bot to kick off ARM API Review.  Missing to check this box in the following scenario may result in delays to the ARM manifest review and deployment.
  - Adding a new service
  - Adding new API(s)
  - Adding a new API version
    -[ ] To review changes efficiently, ensure you copy the existing version into the new directory structure for first commit and then push new changes, including version updates, in separate commits. You can use OpenAPIHub to initialize the PR for adding a new version. For more details refer to the [wiki](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/208/OpenAPI-Hub-Adding-new-API-version).

- [ ] Ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them.

### Breaking Change Review Checklist
If you have any breaking changes as defined in the [Breaking Change Policy](https://aka.ms/AzBreakingChangesPolicy/), request approval from the Breaking Change Review Board.

**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Additional details on the process and office hours are on the [Breaking Change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).

NOTE: To update API(s) in public preview for over 1 year (refer to [Retirement of Previews](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37683/Retirement-of-Previews))

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

adarce committed:
Clarify AKV's description for the maxresults parameter. (#21697)

ventifus committed:
Azure RedHat OpenShift 2022-09-04 (#21248)

* Previous API Version 2022-04-01

* add new api 2022-09-04

refactor list install versions

* Add patterns, remove 201s for PUTs

* Update examples

* Remove patterns, add back 201s for PUTs

* Fix response codes

* Remove patterns

* Update examples

* Update readme.python.md

* Removed 201 modified from added resources

* Satisfy ModelValidation.

* Implement pattern restrictions on new resources

Co-authored-by: Ross Bryan <robryan@redhat.com>
Co-authored-by: Yuchao Yan <yuchaoyan@microsoft.com>

ruowan committed:
add RPSaaS comments (#21801)

andrew-lee-1089 committed:
[Hub Generated] Publish private branch 'ssivathas/Microsoft.MobileNetwork-2022-11-01' (#21752)

* add or modify files

* Use "5Qi" as the schema to define Default QoS Flow 5G QoS Indicator value. (#9817)

On SIM policy we use the 'Default QoS Flow 5G QoS Indicator value', we were incorrectly using the '5QiPriorityLevel' schema (which is an interger between 1 and 127 as per https://github.com/Azure/azure-rest-api-specs-pr/blob/RPSaaSMaster/specification/mobilenetwork/resource-manager/Microsoft.MobileNetwork/stable/2022-11-01/ts29571.json#L117. We should have used 5Qi - an integer between 0 and 255. We made a mistake when we first p ut this in and have no spotted it.

Even though changing the allowed values for a property is a breaking change, we think this is acceptable because:
1) This is our first stable API, there is no pre-existing stable API we need to be back-compatible with
2) 2022-04-01-preview to 2022-11-01 already contains breaking changes that we have discussed in https://eng.ms/docs/strategic-missions-and-technologies/strategic-missions-and-technologies-organization/azure-for-operators/packet-core/private-mobile-network/azure-private-5g-core/cross-team/releases/2022-11-01-arm-api/back-compatibility
3) We have one customer who may need to use values 128-255 -> if that customer does PUTs with the 2022-11-01 API then a GET with the 2022-04-01, it will fail, while this is sad, we think this is an acceptable risk.

* Update ICCID definition to support shorter ICCID (#9910)

GSM 11.1 Version 8 has been replaced by 3GPP TS 11.11, which itself has been replaced by 3GPP TS 102.211 Rel 16, and all of them share the same information for the definition of ICCID.

ISO/IEC 7812-1 2017, ITU-T E.118 - 05/2006 and 3GPP TS 102.211 all alllow ICCID/Primary Account Numbers which are shorter than 19 digits.

While ISO/IEC 7812-1 2017 and ITU-T E.118 - 05/2006 both require a maximum number of digits of 19, and GSM 11.11 is clear than 20 digits ICCID are allowed to be grandfathered in.

ITU-T E.118 - 05/2006 mandates that all new SIM cards use a prefix of 89, but allows 66 as a transitional mechanism for older SIM cards.

ISO/IEC 7812-1 2017 mandates a minimum length of 10 digits. No other specification comments on a minimum size, so we go with this.

The original validation was based on an incorrect reading of the specs, where 19 and 20 were assumed to be the only valid lengths, and the comment on the transitional mechanism was missed.

Even though changing the allowed values for a property is a breaking change, we think this is acceptable because:

1) This is our first stable API, there is no pre-existing stable API we need to be back-compatible with

2) 2022-04-01-preview to 2022-11-01 already contains breaking changes that we have discussed in https://eng.ms/docs/strategic-missions-and-technologies/strategic-missions-and-technologies-organization/azure-for-operators/packet-core/private-mobile-network/azure-private-5g-core/cross-team/releases/2022-11-01-arm-api/back-compatibility

3) We have customers who are currently blocked and having to work around this restriction, and it's acceptable for them to be unable to resubmit the data on the old API.

4) This is relaxing the restriction on a optional field, so the impact of this change is restricted anyway.

5) Not making this change leaves the product ill defined for ever.

* Remove reference to 2021-04-01-preview API in readme

* Fix python output-folder

* Update readme.python.md

Co-authored-by: Richard Whitehouse <github@richardwhiuk.com>
Co-authored-by: Andrew Lee <andrewlee@microsoft.com>
Co-authored-by: Yuchao Yan <yuchaoyan@microsoft.com>

Juliehzl committed:
Release app microsoft.app 2022 10 01 (#21683)

* Adds base for updating Microsoft.App from version preview/2022-06-01-preview to version 2022-10-01

* Updates readme

* Updates API version in new specs and examples

* Add AppState and LatestReadyRevisionName (#21034)

* Add new props

* add defaults

* Add swagger for client cert & CORS policy (#21126)

* Fix password format of env domain for 2022-10-01 (#21463)

* fix

* fix

* Remove AppState property as we decided to postpone it (#21483)

* Remove AppState property as we decided to postpone it

* Fix swagger issues

* More swagger fixes

* s360 swagger correctness fixes (#21472)

Co-authored-by: Nan Jiang <naji@microsoft.com>

* Add kind for managed environment (#21589)

* add

* fix

* fix (#21730)

* fix (#21747)

Co-authored-by: p-bouchon <107427816+p-bouchon@users.noreply.github.com>
Co-authored-by: Ruslan Yakushev <1664475+ruslany@users.noreply.github.com>
Co-authored-by: zhenqxuMSFT <zhenqxu@microsoft.com>
Co-authored-by: najian <jiangnan8924@gmail.com>
Co-authored-by: Nan Jiang <naji@microsoft.com>

najian committed:
Allow edit for daprAIInstrumentationKey (#21815)

Co-authored-by: Nan Jiang <naji@microsoft.com>

qianliu-pdx committed:
[Hub Generated] Review request for Microsoft.SqlVirtualMachine to add version preview/2022-08-01-preview (#21422)

* Adds base for updating Microsoft.SqlVirtualMachine from version preview/2022-07-01-preview to version 2022-08-01-preview

* Updates readme

* Updates API version in new specs and examples

* Adding 2022-08-01-preview version for Sql Virtual Machine swagger

* move all api-version to 2022-08-01-preview

* fix the following validation errors:

R4041 - XmsIdentifierValidation
R4009 - RequiredReadOnlySystemData

* add location for 202 reponse

* fix typo

* fix LRO_RESPONSE_HEADER validation error

* fix Swagger PrettierCheck

* fix 1047 - XmsEnumChanged, manually move IdentityTypeWithNone to
IdentityType

* fix DefinitionsPropertiesNamesCamelCase, adding
azureADAuthenticationSettings to the custom-words.txt

* adding azureADAuthenticationSettings in the alphabetical position.

* fix PostOperationAsyncResponseValidation and  LroLocationHeader errors

* fix typo

* missing quote

* adding quote

* fix DefinitionsPropertiesNamesCamelCase

* remove azureADAuthenticationSettings

* fix ResourceNameRestriction

* use \" to escape "

* fix swagger regex syntax difference comparing to C# regex

* addressing one comment from ARM team to add error response schema for
the default response:

* fix 202 response:
1. adding Location header
2. adding specify x-ms-long-running-operation-options

* UnHealthyReplica to UnhealthyReplica
uuHealthyReplica to unhealthyReplica

* TroubleShoot to Troubleshoot
troubleShoot to troubleshoot

* "UTC format" to "UTC timezone"

* SqlVmTroubleshoot to SqlVmTroubleshooting

* Successfully started SQL virtual machine troubleshoot to Successfully
started SQL virtual machine troubleshooting and similar cases

* "SQL Iaas Extension" to "SQL IaaS Extension"

* Assessment to "an assessment" and other similiar issues

* resolve conflict for custom-words.txt

* resolve conflict

* adding default for troubleshootingScenario

* refactor all sqlVirtualMachineName into common definition and reference
it elsewhere

* add x-ms-long-running-operation-options with azure-async-operation

* remove Location header from the Post request

* add back the Location header for the Post method, as the validation
checks it

* add response schema to HA troubleshooting endpoint 200 response.

* add "default" to false for SQL IAAS auto upgrade property

* renaming:
troubleshootStatus => troubleshootingStatus
unhealthyReplicaProperties => unhealthyReplicaInfo

* update the HA troubleshooting example after the renaming in this
endpoint

* rewording Assessment as SQL best practices Assessment after consulting
with PM.

* revert custom-words.tx file

Co-authored-by: Qiang Liu <qianliu@microsoft.com>

gaoyp830 committed:
Update billingbenefits.json (#21766)

kazrael2119 committed:
update files (#21817)

TimLovellSmith committed:
Scrub custom-words.txt for non-words and descriptions for more inclusive terminology (#21069)

* Delete term AADP from custom-words as its not a commonly known acronym.

* Delete term 'accountid' from custom-words.txt. You should use proper casing and spacing according to context, such as 'AccountId', 'accountId', 'accountID', 'account ID', or so on, instead.

* Delete a whole lot of terms that don't seem to be used anywhere in the specification folder according to a cursory search.

* Delete many duplicates from custom-words.txt

* Use cSpell.json to quarantine away some misspelled words from custom-words.txt. And remove duplicates of PPSX and PPTM.

* 'Mwarev' should not be a word. Terms like 'VMwarevSphere' 'vmware', 'vSphere' should be words instead.

* Using cSpell and custom-words better. 'targetv' should not be a custom word in order to allow 'targetvCenterId'.

* Actually remove UNKNOWON from custom-words now that it has a cSpell.json exception.

* Updation is not a real word.

* Sort 'vcenters'

* Expand away the nonstandard capitalized acronym Vrops from old documentation,

* Noreuse isn't a proper word, so making it a cSpell exception.

* Sort 'cadl'

* Retire some references to whitelists from documentation.

* Replace */* with **

Commit:
CloudServiceRP 2022-09-04 version (#21630)

* create 2022-09-04 cloudService Folder

* update references to new version

* update readme and fix common.json divergence

* Adding zones, example and renaming IP to Ip in cloudservices swagger (#21232)

* Adding zones, example and renaming frontendIPConfigurations to frontendIpConfigurations, LoadBalancerFrontendIPConfiguration to LoadBalancerFrontendIpConfiguration, LoadBalancerFrontendIPConfigurationProperties to LoadBalancerFrontendIpConfigurationProperties

* resolving CI validations

* changing privateIPAddress to privateIpAddress, publiCIPAddress to ppublicIpAddress

* reverting publicIpAddress, privateIpAddress to privateIPAddress, publicIPAddress

* removing x-ms-client-flatten and parameter order change

Co-authored-by: Theodore Chang <theodore.l.chang@gmail.com>
Co-authored-by: ashvermamsft <70934942+ashvermamsft@users.noreply.github.com>

kazrael2119 committed:
update typescript.readme.md in recoveryservicessiterecovery (#21809)

* update typescript.readme.md in recoveryservicessiterecovery

* update file

msyache committed:
Updates for Face PersonDirectory (#21831)

* Updates for Face PersonDirectory

Add parameter descriptions, and add flatten flags for currently
nested request body.

* Another description update

* More

jordan13game committed:
update api scenario schema (#21836)

Co-authored-by: Tianxiang Chen <tianxchen@microsoft.com>

moadus committed:
[Hub Generated] Publish private branch 'dev/monica/nv' (#21847)

* TASK 2104196: Adding a new API version to AzureArcData

In this commit, I just copied over all the current files in
2022-03-01-preview to 2022-06-01-preview so that it serves as a
base commit.
I will add all new files on top of this commit to make main branch
at par with RPSaaSMaster.

* Added the new changes on top of base version

* Updated the readme file

ruowan committed:
don't close and mark PR as stale for customer-reported PR (#21837)

* don't close and mark PR as stable for customer-reported PR

* Update fabricbot.json

12VamshiKrishna committed:
Initial Commit (#21749)

* Initial Commit

* Example Changes

* Parameter Changes

* Parameter Changes

* Definition Changes

* Semantic Fix

* Lint error

Co-authored-by: VamshiKrishna Chirra (Quadrant Resource) <vamshikrishna.chirra@quadrantresource.com>

ruowan committed:
add label for arcReview (#21851)

manikprabhu1208 committed:
[Hub Generated] Publish private branch 'users/prjayasw/agrifoodrp' (#21583)

* Adds base for updating Microsoft.AgFoodPlatform from version preview/2020-05-12-preview to version 2021-09-01-preview

* Adds base for updating Microsoft.AgFoodPlatform from version preview/2020-05-12-preview to version 2021-09-01-preview

* Updates readme

* Updates API version in new specs and examples

* Private endpoint changes

* updated examples

* updated db update example

* build fixes

* changed common-types version

* pr comments

* minor update

* Removed 202 response code from Put call

* update examples

* prettier fix

* enum example fix

* Incremental change of sensor feature

* Adding support for operationResults api for farmbeats resource.

* Making cosmetic changes to fix checks

* Update readme.python.md

pin the title for python SDK

* Fixing one change

* Rsolving comments

* FarmBeats Solution API Review

* fixed error

* pr comment change

* fixing errors

* fixing semantic validation errors

* example file changes

* fixing errors

* pr update fields

* prettier fix

* solution api changes updated

* revert changes in custom words file

* Swagger Spell check fix

* new changes updated in solution definitions

* solution discoverability example file change

* changes updated

* changes updated

* changes updated

* comment changes

* rebaseing main branch

* adding solutions resource in swagger

* rebasing main branch

* solutions resource added in paths

* Solution properties added in definitions

* solution Id added in parameters

* model validation error fix

* model validation errors fixes

* changes updated

Co-authored-by: Prabhav Gupta <prabhavgupta1996@gmail.com>
Co-authored-by: Prabhav Gupta <prabgupt@microsoft.com>
Co-authored-by: Priyanka Jayaswal <prjayasw@microsoft.com>
Co-authored-by: Yuchao Yan <yuchaoyan@microsoft.com>

shavital7 committed:
Remove format:arm-id from v4 since api validation service fails to load the spec, causing global outages for RPaaS RPs (#21860)

juliawelborn committed:
Update RoleManagementAlerts paths (#21735)

* Start updating paths

* update all paths and examples

* Change patch operations to 204 instead of 200

* Update operations paths

Sedols committed:
Adding 2022-12-01-preview version with CMK support (#21741)

* baseline commit for 2022-12-01-preview

* Adding encryption property to Account

* Add Azure-AsyncOperation header

* Fixing Azure-AsyncOperation with Location

mvvsubbu committed:
Fixing Access Connector / Workspace Swagger Correctness (#21783)

* Fixing Access Connector Swagger Correctness

* Addressing Swagger Correctness Issues

* Fixing diskEncryptionSetId readOnly attribute

ruowan committed:
update arc review comments and mail (#21867)

* update arc review comments and mail

* Update .github/comment.yml

Co-authored-by: Ray Chen <raychen@microsoft.com>

kazrael2119 committed:
update typescript.md in network (#21873)

12VamshiKrishna committed:
Initial Commit (#21848)

* Initial Commit

* Prettier Check

Co-authored-by: VamshiKrishna Chirra (Quadrant Resource) <vamshikrishna.chirra@quadrantresource.com>

ruowan committed:
Update sla.yml (#21884)

dosegal committed:
Release securityinsights microsoft.security insights 2022 12 01 preview resolved (#21853)

* Adds base for updating Microsoft.SecurityInsights from version preview/2022-11-01-preview to version 2022-12-01-preview

* Updates readme

* Updates API version in new specs and examples

* Resolve merge conflicts in 2022-12-01-preview (#21772)

* Adds base for updating Microsoft.SecurityInsights from version preview/2022-11-01-preview to version 2022-12-01-preview

* Updates readme

* Updates API version in new specs and examples

* Add tasks API to incidents and tasks action to automationRules (#21542)

* Use generated incidents swagger instead of manual

* try to fix prettier

* try to fix prettier

* try to fix prettier

* fix example casing

* fix example casing

* fix example path and body

* fix alignment with previous swagger

* Add tasks to incidents API

* add task action to automationrules API

* prettier

* fix lint - missing workspaceName pattern

* fix lint - missing workspaceName pattern

* Update entity additional properties to match new oav example validation

* Delete redundant examples

Co-authored-by: ityankel <47292630+ityankel@users.noreply.github.com>

* Fix model and spell validations (#21857)

* Fix model validation

* Add Ueba to custom-words

Co-authored-by: Anat Gilenson <53407600+anat-gilenson@users.noreply.github.com>
Co-authored-by: ityankel <47292630+ityankel@users.noreply.github.com>

stevemunk committed:
Sm data registry api 2022 12 01 preview review (#21823)

* Update dataregistry.json

* Update dataregistry.json

* Update dataregistry.json

* Update dataregistry.json

* Update specification/maps/data-plane/DataRegistry/preview/2022-12-01-preview/dataregistry.json

Co-authored-by: Yenru Chin <danielthank@gmail.com>

* Update dataregistry.json

Co-authored-by: Yenru Chin <danielthank@gmail.com>

junjun99 committed:
[Hub Generated] Publish private branch 'dev/armohommed/200' (#21820)

* add or modify files

* fix model

bexxx committed:
Fix API version in swagger info for OpenAI (#21886)

dorothyhu committed:
[Hub Generated] Review request for Microsoft.HybridCompute to add version stable/2022-11-10 (#21528)

* Adds base for updating Microsoft.HybridCompute from version preview/2022-08-11-preview to version 2022-11-10

* Updates readme

* Updates API version in new specs and examples

* added headers for 202

* add extensions back

* updated description

* changed every v2 to v3 type

dsouzaarun committed:
Service Bus API version 2022-10-01-preview (#20725)

* Base Checkin

* update API version to 2022-10-01-preview

* Adding the partition count property

* updated readme files

* Changed the name of the partitions property to premiumMessagingPartitions

* added headers - azure-asyncoperation

* revertd changes for "modelAsString": true

* revertign the modelasstring changes

* ci Fixes

Co-authored-by: REDMOND\ardsouza <ardsouza@microsoft.com>
Co-authored-by: Ajit Maruti Navasare (MINDTREE LIMITED) <v-ajnav@microsoft.com>
Co-authored-by: Ajit Navasare <ajitnavasare@microsoft.com>

shreyks committed:
Updating Examples for SAP Trans (#21874)

* Updating examples

* Updated payload example

* Updating payloads

* Updating descriptions

Co-authored-by: Shreyas Kottukkal <skottukkal@microsoft.com>

avjerafi committed:
[Hub Generated] Review request for Microsoft.Insights to add version preview/2022-08-01-preview (#21339)

* Adds base for updating Microsoft.Insights from version stable/2022-06-15 to version 2022-08-01-preview

* Updates readme

* Updates API version in new specs and examples

* Added Managed Identity field

* Added publicNetworkAccess field for NSP

* Added ruleResolveConfiguration field for TTM

* Added missing apis to default tag

* Updated package-composite-v1, added package-2022-08-01-preview-only tag

* Fixed swagger path in package-composite-v1

* Reverted change to package-composite-v1

Co-authored-by: Avi Jerafi <avjerafi@microsoft.com>

krishna1s committed:
Loadtest stableapis (#21672)

* publishing stable api version 2022-11-01 for load test service

* prettier run

* spell check

* changing swagger back to 2.0

* adding description

* prettier changes

* renaming opertation  id

* adding x-ms-enum extension

* validation fixes

* renaming examples

* review changes

* adding x-ms-parameter-location

* review fixes

* removing required field

* adding x-ms-enum values

* prettier fix

* spell fixes

* prettier fix

* adding new metrics and app component apis

* updating examples

* removing list resource type api

* changing to kebab case

* changing x-ms-parameter-location value for sdk

* updating example

* removing resourceId

* removing resource Id from examples

* adding missing desc

* removing read only properties from examples

* spell fix

* review fixes

* mode renaming

* properties changes in examples

* renaming

* Update loadtestservice.json

* renaming properties

* renaming property

* desc changes

* Add readme SDK changes (#21315)

Co-authored-by: Harshan B <harshanb@microsoft.com>

* review changes

1. adding file with application octet stream
2.taking fileName as param instead of fileId
3. adding new api for metrics dimension values
4. simplifying metrics api

* model validation fixes

* spell fixes

* updating desc for search and orderby

* desc changes

* adding 201 response

* adding 201 status code

* sdk consistency changes

* filename renaming

* renaming file info

* adding default pass fail criteria action value

* changing operation id and adding x-ms-pageable attributes

* adding Test or Test Run in operation id

* updating operation id

* adding test suffix in operation id

* removing the test suffix

* changing operation id

* changing example response

* updating operation ids

Co-authored-by: Krishna Chandak <krchanda@microsoft.com>
Co-authored-by: Harshan B <Harshan01@gmail.com>
Co-authored-by: Harshan B <harshanb@microsoft.com>

alec-baird committed:
[Hub Generated] Review request for Microsoft.DesktopVirtualization to add version preview/2022-10-14-preview (#21700)

* Adds base for updating Microsoft.DesktopVirtualization from version stable/2022-09-09 to version 2022-10-14-preview

* Updates readme

* Updates API version in new specs and examples

* Added removed changes from 2022-04-01-preview to 2022-10-14-preview

* Removed MigrationRequest from version

* Copied examples over from previous preview

* Added missing changes for private link to swagger

* Added enabled/disabled status to check descriptions.

* Added regex pattern to parameter in types.json

* reverted mistaken common type edit and added pattern to parameters

* Revert "Added enabled/disabled status to check descriptions."

This reverts commit ad177a4e36b3b42d6ac4eb37c622225de3bf95ec.

* Removed pattern requirement; will get exception on linter errors

* Revert "Revert "Added enabled/disabled status to check descriptions.""

This reverts commit 40ef1db774609e7440298a3d236c755aded86942.

* Replaced v1 and v2 common types with v3 and added 201 responses as directed from PR review.

* Only added v3 to new resources to avoid breaking changes

* Fixed incorrect find replace from param to def

* changed system data from v3 to v2 to avoid breaking changes

* Updated resource version from v3 to v1 to avoid breaking changes

* Removed added 201 responses

* Changed PrivateLinkResourceListResult to use internal definition and added pagination to PrivateLink collection calls

* Update to use same version of PrivateEndpointConnection type

* Update PrivateLink version from v3 to v2 to use the same Resource def

* Reverted PrivateLinkConnection v3 to v2 due to sdk problems.

* Added missed pagable to privatelink ListByHostpool

anjorsh committed:
[Hub Generated] Review request for Microsoft.DataProtection to add version preview/2022-11-01-preview (#21441)

* Adds base for updating Microsoft.DataProtection from version preview/2022-10-01-preview to version 2022-11-01-preview

* Updates readme

* Updates API version in new specs and examples

* Added API changes for Blob Backup

* Updated class descriptions based on the feedback

* Fixing spellcheck errors

* Removing "timedout" from custom-words.txt and adding to cSpell.json

vidit-msft committed:
Vidit msft/update database api version v5 (#21878)

* update database api version to 2022-05-01-preview

* rearrange

* rearrange specs file

omrodrig committed:
Fixed the allowed Column Types (#21864)

* Fixed the allowed Column Types

* Fix the formatting of description for enum

mvvsubbu committed:
Adding User Assigned MI Support for Access Connector Resource (#21765)

* Adding new API version for accessConnector

* Adding User Assigned MI Changes

* Updating Version in Examples folder

ambrahma committed:
Review request for Microsoft.DBforPostgreSQL Flexible server to add version stable/2022-12-01 (#21803)

* Adds base for updating Microsoft.DBforPostgreSQL from version preview/2022-03-08-preview to version 2022-12-01

* Updates readme

* Updates API version in new specs and examples

* Fix readme file for new api version change

* Move common-types.json to types folder under postgresql/resource-manager

* Fix path for common-types.json

* Introduced new enum properties for authConfig object instead of previously used enums, based on ARM feedback on earlier 2022-03-08-preview api version PR

* Add LroLocationHeaders for 202 calls

* Updated readme files for go, python and typescript to include new ga api version 2022-12-01

* Multi api version for PG flexible server to inlude 2022-12-01 ga api version

* remove 2022-1201- from multi api in readme.md file

* Update readme files to keep 2022-12-01 ga api version as default version

* Revert changes done to readme.go.md

yangyuan committed:
Cognitive Services 2022-12-01 (#21716)

* Adds base for updating Microsoft.CognitiveServices from version stable/2022-10-01 to version 2022-12-01

* Updates readme

* Updates API version in new specs and examples

* Cognitive Services 2022-12-01

* adress comments

* Revert v4 common types to v2

* Add supression

* Update specification/cognitiveservices/resource-manager/readme.md

Co-authored-by: Jianye Xi <59603451+jianyexi@users.noreply.github.com>

Co-authored-by: Jianye Xi <59603451+jianyexi@users.noreply.github.com>

mvvsubbu committed:
Revert "Adding User Assigned MI Support for Access Connector Resource (#21765)" (#21923)

This reverts commit 2181a44b0f75c71ae65142702876e3de59041f3a.

meirloichter committed:
[S360 fix] update Agreements.json to reflect the current api (#21844)

* update Agreements.json to reflect the current api

* fix prettier issues

* fix prettier issues

matanova8 committed:
[Hub Generated] Publish private branch 'dev-azure-kusto-Microsoft.Kusto-2022-11-11' (#21802)

* Adds base for updating Microsoft.Kusto from version stable/2022-07-07 to version 2022-11-11

* Updates readme

* Updates API version in new specs and examples

* changes from Kusto monolith

* "type": "object",

* Format

* 1. Add "_List"
2. ApiVersionParameter from common types
3. Add required state "Canceled"

* Location_Skus_List to Skus_List

* Typo fix: "an CosmosDb" to "a CosmosDb"

* Support zoneDetails

* Revert "Support zoneDetails"

This reverts commit d873727f042d8ff2b6624bfc8049653fa84ef72d.

* zoneDetails to 2022-11-11

* "type": "object" + "x-ms-identifiers": [
                               "name"
                             ],

raosuhas commented:
What is this PR for it looks like you ae trying to get a number of files from various namespaces into main ? PLease fix the PR to only include what is required and re-submit

dzeliar committed:
[Communication - Telephony Service] Trunk status API (#19233)

* trunkstatus 2022-07-01 preview

* fix nextLink

* remove pageable, fix sdk typo

* add port to trunk properties

* replace by health endpoint

* fix overallHealth property

* fix enums

* update enum and add descriptive properties

* enums to camelcase

* remove trunkstatus folder

* add health to trunk properties

* create new preview

* address review comments

* fix example

* the latest api version swaggers are in the default tag

* remove unused response codes from example

* fix prettier

* update python and ruby tag to the latest version

* Unify naming SBC->Trunk, move Extended to beginning of Trunk and SipConfiguration to suggest inheritance

* bump up version

* bump up the swagger version

* fix errors missing enabled property

* revert error code version back to new

* fix typo and duplicate definition in previous reamdme versions

* fix typo

* address comments

* update ping description

* shuffle object definition to help APIview render document properly

* Revert "shuffle object definition to help APIview render document properly"

This reverts commit a682d6a26f93866ca3467253168be5c569e958a6.

* remove redundant type:object on

moshemal committed:
Momalka/fix examples (#21889)

* change moniotor account id

* add lables to recording rule

Co-authored-by: Moshe Malka <momalka@microsoft.com>

raych1 committed:
Added two services PR assignment (#21901)

* Added two services PR assignment

* Added kusto service for PR assignment

* Remove rpsaasdev branch review for voiceservices

navba-MSFT committed:
{AzureRecoveryServices} fixes Azure/azure-sdk-for-python#27977 recovery_point_id property description (#21919)

* {AzureRecoveryServices} fixes Azure/azure-sdk-for-python#27977 recovery_point_id property description

 fixes Azure/azure-sdk-for-python#27977 `recovery_point_id` property description

The current description for `recovery_point_id` property says "ID of the backup copy to be recovered." This is very confusing. Users might confuse it for the resource ID. The description should be made clear that it is "Recovery Point ID Name of the backup copy to be recovered."

* Updating the recent stable version

* Update bms.json

apmehrotra committed:
Dev hybridconnectivity microsoft.hybrid connectivity 2023 03 15 (Draft PR, please review it, but do not merge it yet) (#21631)

* Adds base for updating Microsoft.HybridConnectivity from version preview/2022-05-01-preview to version 2023-03-15

* Updates readme

* Updates API version in new specs and examples

* Adding upcoming changes for GA

* Adding service configuration

* Adding serviceConfigurationToken

* Changing service configuration type

AntonioVT committed:
[Manual PR] Review request for Microsoft.ContainerRegistry to add version stable/2022-12-01 (#21807)

live1206 committed:
Revert "Dev hybridconnectivity microsoft.hybrid connectivity 2023 03 15 (Draft PR, please review it, but do not merge it yet) (#21631)" (#21974)

This reverts commit f9e272186af02eb26b2563e438e8b120c772a5cc.

rkday-pro committed:
[Hub Generated] Publish private branch 'voiceservice-private-preview' (#21862)

* add or modify files

* Sync custom-words to main

* Add custom words

kazrael2119 committed:
update typescript.md in service-map (#21961)

* update typescript.md in service-map

* update files

neeerajaakula committed:
[Hub Generated] Review request for Microsoft.KeyVault to add version preview/2021-06-01-preview (#21671)

* Fix s360 alerts for properties being returned in GET but not exising in PUT requests

* Prettier fix

* Fix model validation error for long running operation response missing location header

* removed default value for array as it breaks the arm template schema creation

* removed default value for array for managed hsm virtual network rule as it breaks the arm template schema creation

yanjungao718 committed:
add pbi workspace linked service (#21888)

catalinaperalta committed:
[AnomalyDetector] Update CADL specification (#21969)

* update mvad enums

* update uvad enums

* update main.cadl and package.json

* update cadl-project.yaml

* update package.json

* update swagger

JoshLove-msft committed:
Add a note to Event Grid readme about review (#21968)

* Add a note to Event Grid readme about review

* Update specification/eventgrid/data-plane/readme.md

Co-authored-by: Laurent Mazuel <lmazuel@microsoft.com>

Co-authored-by: Laurent Mazuel <lmazuel@microsoft.com>

kazrael2119 committed:
add typescropt.md (#21981)

* add typescropt.md

* update

* update

* update

Commit:
ASE VM Marketplace (#21890)

* Base Commit

* New api's

* Review comment fixes

dindurkhya committed:
Updating Examples for Create flow with proper values (#21839)

* Updating Examples for Create flow with proper values

* Updating Examples for Create Infra

* Adding SAP Trans and DIsk Configuration common example

* Adding Install Example payloads for HA systems

tadelesh committed:
Update pull_request_assignment.yml (#21985)

* Update pull_request_assignment.yml

* Update pull_request_assignment.yml

oravital7 committed:
[Hub Generated] Publish private branch 'oravital-update--dev-security-Microsoft.Security-2022-01-01-preview' (#21988)

* Update governance operation result

* Fix Model Validation

* Fix comment

* Fix comment

FumingZhang committed:
update custom words (#21929)

FumingZhang committed:
[Hub Generated] Publish private branch 'containerservice/official/v20221218' (#21987)

* [AutoSync] 6a6ad0e3336 Merged PR 7353698: [API Release] Sync swagger changes in GitHub PR #21069

* [AutoSync] c7b8955aefd Merged PR 7359461: [API Release] Fix Swagger LintDiff errors

* [AutoSync] 76fc4392c08 Merged PR 7379598: [Swagger][2022-11 APIs] azure auth-provider kubeconfig is fully removed in v1.26 instead of v1.25

Co-authored-by: swagger-automation <swagger@microsoft.com>

dsouzaarun committed:
Event Hub API version 2022-10-01-preview (#20724)

* Base Checkin

* Replace API version with 2022-10-01-preview

* Adding settings for Compaction

* Fixed examples

* Adding new api version to README.md

* Getting PolicyRetention enum to be modelled as string

* Adding latest api as default

* Fixing LintDiff

* Fixing Model Validation

* updated readme files

* Splitting the RetentionTimein Hours into two seperate properties.

* lint fix

* fixed typo

* Revert "fixed typo"

This reverts commit 2387ca3bf6f02e3a2df4966de2234c9dd2212e2b.

* Revert "lint fix"

This reverts commit 2c775034f650404cb9fe85bcc1730cb8425847b5.

* Revert "Splitting the RetentionTimein Hours into two seperate properties."

This reverts commit caf5d9ebaf2d5429ce489ba8004e805e5bd9159c.

* Fixing examples

* removed the supperssion

* added headers - azure-asyncoperation

* fixing the CI failures

* Revert "added headers - azure-asyncoperation"

This reverts commit 2d60d5c0d8d7839da3e6cf2d5f1aeba90c1efacb.

* updated CI fixes

* headers fix for cluster catch

Co-authored-by: REDMOND\ardsouza <ardsouza@microsoft.com>
Co-authored-by: davadhani <davadhani@microsoft.com>
Co-authored-by: Ajit Maruti Navasare (MINDTREE LIMITED) <v-ajnav@microsoft.com>
Co-authored-by: Ajit Navasare <ajitnavasare@microsoft.com>

sanjaiganesh committed:
Update Data container ubuntu os version (#21995)

* Add blockchain to latest profile

* Add additional types

* update

Co-authored-by: Mark Cowlishaw <markcowl@microsoft.com>

rishikaparashar committed:
[Hub Generated] Review request for Search to add version preview/2022-09-01-preview (#21898)

* Boundary Documentation changes in Dev maps search 2022 09 01 preview (#21892)

* Adds base for updating Search from version preview/2022-02-01-preview to version 2022-09-01-preview

* Updates readme

* Updates API version in new specs and examples

* Rename geocoding.json to search.json

* Added Boundary service APIs

* Fixed references to common spec

* Use an array instead of a map for geometries copyright

* Remove readOnly attribute from responses and default/max for the top parameter

* Mark seach/polygon location parameter as required

* Rename entityType parameter to resultType

* Rename entityType parameter to resultType in the example too

* Use View parameter from Common spec

* Doc Updates:1. Updated 'location' request parameter to 'coordinates' and updated the GetPolygon example with the correct values 2. Updated the boundary description to mention the expected geometry objects in the boundary API response 3. Updated View description to mention link to azure maps supported views details.

* Update custom-words.txt

Added POIs word to fix spell check error.

Co-authored-by: Igor Shishkin <specialforest@users.noreply.github.com>
Co-authored-by: Igor Shishkin <igshishk@microsoft.com>

* Update common.json (#21897)

* Updated the Boundary parameters description. (#21913)

* Updated the Boundary description and removed pois from custom words list.

* spelling validation fix for POIs - adding POIs to the custom-words list.

* Updated POIs description.

* addressed PR comment - Updated View description.

* Updated the Boundary view parameter description - default value set to auto.

Co-authored-by: Igor Shishkin <specialforest@users.noreply.github.com>
Co-authored-by: Igor Shishkin <igshishk@microsoft.com>

12VamshiKrishna committed:
EnableInteractivequery Changes (#21936)

* EnableInteractivequery Changes

* Spell Check

Co-authored-by: VamshiKrishna Chirra (Quadrant Resource) <vamshikrishna.chirra@quadrantresource.com>

chandrasekarendran committed:
Adding new api-version 2022-12-01 to Microsoft.DataProtection (#21661)

* Base commit for updating Microsoft.DataProtection from version stable/2022-05-01 to version stable/2022-12-01

Base commit for updating Microsoft.DataProtection from version stable/2022-05-01 to version stable/2022-12-01

* Updating readme

* Updating api version in new specs and examples

* Updating examples

* Adding CSR related changes

* Adding softDeleteSettings to backupVault

* Adding examples

* fixing swagger

* Fixing readme

* Adding description

* Adding immutability settings

* Update dataprotection.json

Added Expiry time field for RP contract

* Update ListRecoveryPoints.json

Updated ListRP example for expiry Time

* Merging latest fix

* Adding OperationalStore enum value

* Using parameters from v4 common

* Adding parameter for BackupInstanceName

* PatchResourceGaurd Fix

* Adding identity property in PatchResourceGuardInput

* Removing identity in resourceGuard resource

* Removing identity from ResourceGuard patch

Co-authored-by: amchandn <43201777+amchandn@users.noreply.github.com>
Co-authored-by: cvarsh <cvarsh@microsoft.com>
Co-authored-by: Srinivas Charan Madu <smadu@microsoft.com>

manikprabhu1208 committed:
s360 kpi fix (#22013)

msyyc committed:
Create readme.python.md (#21972)

* Create readme.python.md

* Update readme.md

bexxx committed:
Fix required api version parameter in OpenAI (#21921)

* Fix API version in swagger info for OpenAI

* Fix api version parameter of open ai ga version

navba-MSFT committed:
{AzureFrontdoor} fixes Azure/azure-cli#24758 fixing ActionType enum to include AnomalyScoring (#21947)

fixes Azure/azure-cli#24758 fixing ActionType enum to include AnomalyScoring

Running the below command to override the managed rules with --action set to AnomalyScoring fails:
> az network front-door waf-policy managed-rules override add

```
az network front-door waf-policy managed-rules override add: 'AnomalyScoring' is not a valid value for '--action'. Allowed values: Allow, Block, Log, Redirect.

```

This is because currently only the `Allow, Block, Log, Redirect` enums are specified. `AnomalyScoring` value should also be included in the `ActionType`  enums.

thramu committed:
Creating Settings.json spec under preview (#21723)

* Creating Settings.json spec under preview

* Adding settings.json into readme.md

* Prettier fix

* Review comments

Commit:
Feature/cplat 2022 11 01 (#21571)

* add 2022-11-01 folder

* update readme, update version reference

* Added userData property for PUT Restore Point API

Added user data property for PUT Restore Point API

* Revert "Added userData property for PUT Restore Point API"

This reverts commit 94c96d8ca8586b37a59083efd6357ccf2594dd61.

* Reimage Support for Non-Ephemeral VMs (#21090)

* Reimage support for non-ephemeral VMs

* small changes

* small changes

* resolving issues

* pretty check

* pretty install

* remove package-lock

* remove package-lock

* Added userData property in Restore Point PUT API endpoint (#21262)

* Added userData property in Restore Point PUT Operation

* Update based on review comment

* Updated description

Co-authored-by: Ashwani Tiwari <ashwtiwari@microsoft.com>

* Add OS Image profile to scheduled events profile and maxSurge in rolling upgrade policy in Microsoft.Compute (#21253)

* add reimage notification

* update reimage notification, add max surge

* fix grammar

* Adding requireGuestProvisionSignal and constrainedMaximumCapacity mis… (#21433)

* Adding requireGuestProvisionSignal and constrainedMaximumCapacity missing properties

* Moved the property at the end

Co-authored-by: Avaiyang Garg <avaiyanggarg@micrososft.com>

* Add serviceArtifactReference property (#21607)

* Add serviceArtifactReference property

* Moved the property inside the virtualMachineScaleSet file

* Minor change in example

Co-authored-by: Avaiyang Garg <avaiyanggarg@micrososft.com>

* add platformFaultDomain in instanceView (#21605)

* add platformFaultDomain in instanceView

* change platformFaultDomain values

* add APIVersionPattern Suppression (#21675)

Co-authored-by: Theodore Chang <theodore.l.chang@gmail.com>

* Adding imageDeprecationStatus properties to VirtualMachineImage response (#21615)

* Changes to virtualMachineImage.json to include ImageDeprecationStatus properties

* Changes to example json

* Fixed LintDiff errors

* Addressed review comment

* Introduce rollbackFailedInstancesOnPolicyBreach in VMSS RollingUpgradePolicy  (#21676)

* add property in compute.json

* add in 2022 file

* add rollbackFailedInstancesOnPolicyBreach in examples

* Revert "add property in compute.json"

This reverts commit 15a1e10503cc4e34b0b69f351cc8f23339002360.

* remove space

Co-authored-by: Sachin Doddaiah Shylaja <sadoddai@microsoft.com>
Co-authored-by: Theodore Chang <thchan@microsoft.com>

* fix spellingCheck error, LintDiff error, model validation, and typo

* fix modelValidation

* model validation

* update headers for examples

* model validation errors

* model validation

* Revert "model validation"

This reverts commit d3810a7866a9093b6ef75a57834e1aaeb4e139f3.

* Revert "model validation errors"

This reverts commit ba34d1b2af2bef58b60c16e9ed1cda00803a2ce0.

* Revert "update headers for examples"

This reverts commit 216b3dc1367655b8c7867a9d7a772fe7e2fa1583.

* Revert "model validation"

This reverts commit 285345656321e984bc99486a575e35a4d3359667.

* Revert "fix modelValidation"

This reverts commit a5fa156d42130566b827df88d6dea4ee591f4a6c.

* Revert "fix spellingCheck error, LintDiff error, model validation, and typo"

This reverts commit 9f60b71000be72bf4511feefcda0820b570345de.

* fix spelling error CI, file name typo, Lint Diff

Co-authored-by: Theodore Chang <theodore.l.chang@gmail.com>
Co-authored-by: Ashwani Tiwari <ashwtiwari@microsoft.com>
Co-authored-by: srcharug <62158390+srcharug@users.noreply.github.com>
Co-authored-by: csassasin <89677872+csassasin@users.noreply.github.com>
Co-authored-by: John Zhang <116316220+johnzhang-microsoft@users.noreply.github.com>
Co-authored-by: avaiyanggarg <116031789+avaiyanggarg@users.noreply.github.com>
Co-authored-by: Avaiyang Garg <avaiyanggarg@micrososft.com>
Co-authored-by: parahluwalia <117622623+parahluwalia@users.noreply.github.com>
Co-authored-by: RajalakshmiSwaminathan <raswamin@microsoft.com>
Co-authored-by: Sachin Doddaiah Shylaja <sachin.ds92@gmail.com>
Co-authored-by: Sachin Doddaiah Shylaja <sadoddai@microsoft.com>

XiaofeiCao committed:
readme.java.md (#22040)

leniatgh committed:
Check in Appconfig scenarios and recording assets (#21798)

* add appconfig control-plane scenarios

* touch swagger

* push tag successfully

* check in tag

* move assets.json to scenarios folder

* update tag

* add appconfig data-plane scenarios

* push data-plane apitest recording

* update mgmt recording

* restore swagger

* update recording

maddiemlucas committed:
change array name to settings (#21335)

thomasp98296 committed:
Task 16564293: Update swagger description of new budget alert. (#21879)

* Task 16564293: Update swagger description of new budget alert.

* Also remove "budget" as it will not longer be used.

* Add custom words

Co-authored-by: Thomas Pham <thompham@microsoft.com>

dhu5432 committed:
Add additionalProperties to Microsoft.ResourceGraph/resourcesHistory (#21088)

* Additional Properties

* Not anonymous

* PrettierCheck

Commit:
add or modify files (#21989)

navba-MSFT committed:
{AzureAPIM} fixes Azure/azure-rest-api-specs#21849  'in body' parameter specification missing (#21875)

* {AzureAPIM} fixes Azure/azure-rest-api-specs#21849  'in body' parameter specification missing

fixes Azure/azure-rest-api-specs#21849  'in body' parameter specification missing

The CreateOrUpdate operations on both of the following resource types are missing the in body parameter in the OpenAPI specification (OAS):

Microsoft.ApiManagement/service/contentTypes
Microsoft.ApiManagement/service/contentTypes/contentItems
This causes the documentation TryIt and autogenerated libraries (e.g. nodejs @azure/arm-apimanagement) to break, as they don't support passing in the properties in the body, making these resource types unable to be updated or created.

For example, if you run the "Try It" for this API, it doesn't give you a body field in the auto-generated REST HTTP request form. The body field is needed, but it doesn't give you that field as it's not populated in the OAS for this resource type.

See this: https://github.com/Azure/azure-rest-api-specs/blob/7245d0f5cc01bf95df9d7ae893f252d4001b4ad7/specification/apimanagement/resource-manager/Microsoft.ApiManagement/stable/2021-08-01/apimcontenttypes.json#L132-L162

* Update apimcontenttypes.json

Alancere committed:
[Go] add servicenetworking readme.go.md (#22082)

sameerpatro committed:
ASE New API version 2023-01-01-preview Changes (#22072)

* Base Commit from previous released version

* New API version changes

ArcturusZhang committed:
fix cosmosdb examples (#22087)

amperugu committed:
Add new tier and sku - Standard_Basic for Application Gateway (#21976)

* Add new tier and sku for standard_basic

* spaces

Commit:
Add scan folder and spark config support in Sparkjob activity (#21768)

* add scan folder and spark config support

* fix typo

* update description

* add custom word

kristapratico committed:
[Language - QA authoring] correct `source` to required param for update sources (#22065)

* correct source to required param for update sources

* fix validation errors while we're here

leniatgh committed:
Document how to integrate test-proxy (#21811)

* document how to integrate test-proxy

* update doc

* fix comments

guptashweta9 committed:
RAN Info Updated (#22101)

leni-msft commented:
@htippanaboya Could you clarify the purpose of this PR? Is it just to update `release-Microsoft.Network-2022-09-01` branch with main? There're too many changed files, so difficult to review.

raych1 committed:
Adding cadl sample project (#21722)

* Adding cadl sample project

* cadl sample: packaqe.json & cadl-project.yaml (#21903)

* cadl sample: packaqe.json & cadl-project.yaml

* Update cadl-project.yaml

* Use faked shared model

* Update documentation/cadl-sample-project/specification/contosowidgetmanager/Contoso.WidgetManager/cadl-project.yaml

Co-authored-by: Wes Haggard <weshaggard@users.noreply.github.com>

* Update cadl-project.yaml

* Adjusted to minimum file set

* update .Net emitter configuration

* update java emitter config

* Fixed examples issues

* Fixed example style issue

Co-authored-by: Wei Dong <40835867+dw511214992@users.noreply.github.com>
Co-authored-by: Wes Haggard <weshaggard@users.noreply.github.com>

abhinav-ghai committed:
Add 2023-02-27-preview for DigitalTwins (#21953)

* Add 2023-02-27-preview for DigitalTwins

* Update readme.md

raych1 committed:
Updated Toc in cadl structure markdown document (#22118)

IndraneelPal committed:
Changed Ehc Resource Id property to camel case (#22019)

bitsforthought committed:
Removing myself from CODEOWNERS since I no longer work on Azure Search (#22033)

Co-authored-by: Bruce Johnston <brjohnst@microsoft.com>

thatChang committed:
Release botservice microsoft.bot service 2022 09 15 (#22077)

* Base commit from 2022-06-15-preview

* Fix S360 issues

Co-authored-by: Chang Liu <liuchang@microsoft.com>

joseharriaga committed:
Add entry to CODEOWNERS (#22115)

msyyc committed:
update (#22144)

leni-msft commented:
Close this PR due to no response.
